typescript/db/src/repository/label.ts

```diff
@@ -1,77 +1,91 @@
+import { DbLabel, Repository } from "@labelflow/common-resolvers";
+import { LabelWhereInput } from "@labelflow/graphql-types";
+import { isEmpty } from "@labelflow/utils";
 import { Prisma } from "@prisma/client";
-import { Repository, DbLabel } from "@labelflow/common-resolvers";
+import { isNil } from "lodash/fp";
+import { Entry } from "type-fest";
 import { getPrismaClient } from "../prisma-client";
-import { castObjectNullsToUndefined } from "./utils";
 import { checkUserAccessToDataset } from "./access-control";
+import { AuthorizationError } from "./authorization-error";
+import { castObjectNullsToUndefined } from "./utils";
 
-export const countLabels: Repository["label"]["count"] = async (
-  whereWithUser
-) => {
-  if (!whereWithUser) {
-    return 0;
-  }
-  const { user, ...where } = whereWithUser;
-  if (user?.id == null) {
-    return 0;
-  }
+const getIdFilter = (
```

Comment:
Make the function name a bit more explicit about the fact that we are handling a *list* of IDs